catalog/urls.py

```diff
@@ -0,0 +1,6 @@
+from django.urls import path
+from . import views
+
+urlpatterns = [
+    
+]
```

Comment:
thêm line trống cuối file nha
```suggestion
urlpatterns = []
```